### PR TITLE
Warnings cleanup

### PR DESCRIPTION
**Pull request by AsYlum-**
I've fixed all the warnings so after merging we can then adjust github action to treat all ecompile warnings as errors.

Most of the work is removing unused variables. A bit of renaming where local variable covers global one with same name. In situations where variables were really unused I was removing them and in some places I've just commented out code. 

It compiles and runs. Distro still need some testing so I've figured out it will be good moment to get rid of some lazy coding habits. :)

### CODE REVIEW

File: pkg/commands/test/AAAtest.src [hunk 1]
```diff
@@ -10,16 +10,16 @@ use uo;
 include ":attributes:attributes";
 
 program test_cmd(who, text)
+	text := text;
```
Comment (eric-swanson) [inline pkg/commands/test/AAAtest.src:13]:
```suggestion
program test_cmd(who, unused text)
```

File: scripts/ai/combat/spellFireEvent.inc [hunk 2]
```diff
@@ -61,16 +61,17 @@ function in_combat_event_loop(opponent, loops)
 endfunction
 
 function prepare_for_fight(opponent)
+  opponent := opponent; // Avoid compiler warnings
```
Comment (eric-swanson) [inline scripts/ai/combat/spellFireEvent.inc:64]:
```suggestion
function prepare_for_fight(unused opponent)
```

Comment (AsYlum-) [reply scripts/ai/combat/spellFireEvent.inc:64]: Ooh I need to update myself on recent changes in ecompile wold :) Thanks.

Comment (AsYlum-) [reply scripts/ai/combat/spellFireEvent.inc]: Changed all to unused keyword. That is nice addition :)

**Comment by AsYlum-** [comment]
After getting some hints from @eric-swanson I've redo the commits to use unused keyword and not old re-assignment method. I'm sure there are some pieces that still needs updating but for now we have no warnings during compilation.

**Comment by AsYlum-** [comment]
PR updated to recent housing changes.

**Pull request merged**